integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -0,0 +1,272 @@
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.service.bridge.actions.kafkatopic.KafkaTopicAction;
+import com.redhat.service.bridge.infra.api.APIConstants;
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+import com.redhat.service.bridge.infra.models.dto.BridgeStatus;
+import com.redhat.service.bridge.infra.models.filters.BaseFilter;
+import com.redhat.service.bridge.infra.models.filters.StringEquals;
+import com.redhat.service.bridge.infra.models.filters.ValuesIn;
+import com.redhat.service.bridge.infra.utils.CloudEventUtils;
+import com.redhat.service.bridge.integration.tests.common.AbstractBridge;
+import com.redhat.service.bridge.manager.api.models.requests.ProcessorRequest;
+import com.redhat.service.bridge.manager.api.models.responses.BridgeListResponse;
+import com.redhat.service.bridge.manager.api.models.responses.BridgeResponse;
+import com.redhat.service.bridge.manager.api.models.responses.ProcessorResponse;
+
+import io.cloudevents.SpecVersion;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.addBridge;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.createProcessor;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.deleteBridge;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.deleteProcessor;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getBridgeDetails;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getBridgeList;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getProcessor;
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class End2EndTestITStep extends AbstractBridge {
+
+    private static final String TOPIC_NAME = "myKafkaTopic";
+    private static final Set<BaseFilter> FILTERS = Collections.singleton(new StringEquals("source", "StorageService"));
+    private static final String TRANSFORMATION_TEMPLATE = "{\"v1\": \"{data.k1}\"}";
+    private static final String PROCESSOR_NAME = "myProcessor";
+    private static String BRIDGE_NAME = "notificationBridge";
+
+    private static String bridgeId;
+    private static String processorId;
+    private static String endPoint;
+    private static String ingressMetrics;
+    private static ProcessorRequest processorRequest;
+
+    @Given("Manager url is not accessible")
+    public void authenticationIsEnabled() {
+        given()
+                .filter(new ResponseLoggingFilter())
```

Comment:
Logging filters look quite verbose, I would rather omit them by default, can be enabled with some specific Maven flag (in case someone will need to investigate requests/responses). What do you think?

Reply:
Yes i think we can remove logging filters